x/qgb/client/cli/valsetconfirm.go

```diff
@@ -1,19 +1,55 @@
 package cli
 
 import (
+	"github.com/celestiaorg/celestia-app/x/qgb/types"
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/spf13/cobra"
+	"strconv"
 )
 
+func GetQueryCmd() *cobra.Command {
+	//nolint: exhaustivestruct
+	qgbQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the qgb module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	qgbQueryCmd.AddCommand([]*cobra.Command{
+		CmdGetValsetConfirm(),
+	}...)
+
+	return qgbQueryCmd
+}
+
 func CmdGetValsetConfirm() *cobra.Command {
 	//nolint: exhaustivestruct
 	cmd := &cobra.Command{
 		Use:   "valset-confirm [nonce] [bech32 validator address]",
 		Short: "Get valset confirmation with a particular nonce from a particular validator",
 		Args:  cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// TODO
-			return nil
+			clientCtx := client.GetClientContextFromCmd(cmd)
+			queryClient := types.NewQueryClient(clientCtx)
+
+			nonce, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			req := &types.QueryValsetConfirmRequest{
+				Nonce:   nonce,
+				Address: args[1],
+			}
+
+			res, err := queryClient.ValsetConfirm(cmd.Context(), req)
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(res)
```

Comment:
we definitely do not have to do this in this PR or the next few, but eventually, we should include integration tests similar to what we do for the payment module. We might also want to write an issue to remind future us.

Reply:
So no need to have a client module? 
Cool, I can remove this stuff and only add it later when needed.